lab/lab-guide.md

```diff
@@ -34,47 +43,19 @@ Before we implement new functionality in this project, we need to set up authent
 
 ## Configure authentication
 
-To access our Azure resources, we need to be authenticated from our terminal in Visual Studio Code. Then we need to set up the role assignments required to access Azure OpenAI and Azure Cosmos DB.
-
-1. In the same terminal we just opened, change directories to **infra/scripts**.
-
-    ```bash
-    cd ../infra/scripts
-    ```
+To access our Azure resources, we need to be authenticated from our terminal in Visual Studio Code. This application uses authentication with [Microsoft Entra ID](https://learn.microsoft.com/entra/identity/). Role assignments have already been created for your user during resource deployment. If you're already signed in with the user credentials you used for deployment, skip to [Build and run the application for the first time.](#build-and-run-the-application-for-the-first-time)
 
 1. Log in to the Azure CLI by entering the following command in the terminal window.
```

Comment:
Can we update this to `Log in to the Azure CLI by entering the following command in the visual studio code terminal window`

A few people in the lab were doing that in actual terminal and facing some issues. It's better to point it out here again.